test/index.test.ts

```diff
@@ -1,14 +1,14 @@
-import connect = require('connect')
-import request = require('supertest')
+import connect from 'connect'
+import request from 'supertest'
 import { IncomingMessage, ServerResponse } from 'http'
 
-import ienoopen = require('..')
+import ienoopen from '..'
```

Comment:
I'm not sure about this change. I worry that it leaves old behavior untested (in other words, JS users can't do `const ienoopen = require('ienoopen')` as before).

What do you think?

Reply:
Good point, we should aim to match how it's used in user-land. Have reverted. :+1: